mypy/typeanal.py

```diff
@@ -892,9 +902,10 @@ def tuple_type(self, items: List[Type]) -> TupleType:
 
 # Mypyc doesn't support callback protocols yet.
 FailCallback = Callable[[str, Context, DefaultNamedArg(Optional[ErrorCode], 'code')], None]
+NoteCallback = Callable[[str, Context, DefaultNamedArg(Optional[ErrorCode], 'code')], None]
```

Comment:
This alias is identical to the above one. You can rename it to `MsgCallback` and use it to annotate both `fail` and `note`.